cip/CIP2015-06-24-call-procedures.adoc

```diff
@@ -0,0 +1,250 @@
+= CIP2015-06-24 Calling Stored Procedures
+:numbered:
+:toc:
+:toclevels: 3
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Stefan Plantikow <stefan.plantikow@neotechnology.com>
+*Author:* Nigel Small <nigel.small@neotechnology.com>
+*Author:* Petra Selmer <petra.selmer@neotechnology.com>
+*Author:* Rickard Öberg <rickard.öberg@neotechnology.com>
+
+[abstract]
+.Abstract
+
+This proposal introduces stored procedures as a new concept to Cypher and proposes a new facility
+for calling them.
+
+User-defined functions are a separate concern that might be addressed in a future CIP.
```

Comment:
The empty line messes up the formatting of the abstract. Keeping the abstract in one paragraph is probably the right thing here.